### PR TITLE
CP4 EVM compatibility

### PR DESCRIPTION
**Pull request by annamehr**

### CODE REVIEW

File: cps/cp35 [hunk 1]
```diff
@@ -0,0 +1,65 @@
+---
+cp: 35
+title: EVM compatibility on the Centrifuge Chain and Centrifuge App
+authors: annamehr
+contributors:
```
Comment (Rhanok09) [inline cps/cp35:5]: You want to add Jeroen and Frederik like in the RFC on the Forum?

File: cps/cp35 [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+proposal-type: CP4 technical
+date-proposed: 2023-02-24
```
Comment (Rhanok09) [inline cps/cp35:7]: The proposal was created 2023-03-07 on the Forum

Comment (ImdioR) [reply cps/cp35:7]: 1.cp35 - should be CP39
2. the name of the file should be cp39.md
3. status: polling

File: cps/cp35 [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+date-ended: 
+status:
```
Comment (Rhanok09) [inline cps/cp35:9]: Status should be **polling**.

File: cps/cp35 [hunk 4]
```diff
@@ -0,0 +1,65 @@
+---
+cp: 35
```
Comment (Rhanok09) [inline cps/cp35:2]: PR# is 39 so it should be 39.

**Review by ImdioR — APPROVED** [review]

**Pull request merged**